.github/workflows/main.yml

```diff
@@ -2433,7 +2430,7 @@ jobs:
         if: contains(matrix.os, 'ubuntu')
         uses: shivammathur/setup-php@v2
         with:
-          php-version: '7.4'
+          php-version: '8.3'
```

Comment:
@junichi11 I bumped the PHP version so that we don't test on EOL releases - lets see if it works